### ISSUE TITLE
OSQP segmentation fault using direct mode 

### ISSUE THREAD

**Issue by jd-lara** [osqp/OSQP.jl#86]
The issue happens when the model is being displayed into the REPL. 

MWE

```julia
julia> using JuMP, OSQP
julia> m = direct_model(OSQP.Optimizer())
```

Resulting segmentation fault 

```Julia
signal (11): Segmentation fault: 11
in expression starting at none:0
unsafe_load at ./pointer.jl:105 [inlined]
unsafe_load at ./pointer.jl:105 [inlined]
dimensions at /Users/jdlara/.julia/packages/OSQP/BU4WO/src/interface.jl:498 [inlined]
get at /Users/jdlara/.julia/packages/OSQP/BU4WO/src/MOI_wrapper.jl:407
unknown function (ip: 0x1181c08df)
get at /Users/jdlara/.julia/packages/JuMP/qhoVb/src/JuMP.jl:863
num_variables at /Users/jdlara/.julia/packages/JuMP/qhoVb/src/JuMP.jl:430
show at /Users/jdlara/.julia/packages/JuMP/qhoVb/src/print.jl:166
show at ./multimedia.jl:47
unknown function (ip: 0x1181c07f8)
display at /Users/jdlara/.julia/packages/OhMyREPL/07uNa/src/output_prompt_overwrite.jl:8
unknown function (ip: 0x1181bdba8)
display at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:218
display at ./multimedia.jl:328
jl_apply at /Users/julia/buildbot/worker/package_macos64/build/src/./julia.h:1690 [inlined]
do_apply at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:655
jl_f__apply at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:669 [inlined]
jl_f__apply_latest at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:705
#invokelatest#1 at ./essentials.jl:710 [inlined]
invokelatest at ./essentials.jl:709 [inlined]
print_response at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:238
unknown function (ip: 0x1181bd357)
print_response at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:223
unknown function (ip: 0x1181bc465)
do_respond at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:822
unknown function (ip: 0x1181ba690)
jl_apply at /Users/julia/buildbot/worker/package_macos64/build/src/./julia.h:1690 [inlined]
do_apply at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:655
jl_f__apply at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:669 [inlined]
jl_f__apply_latest at /Users/julia/buildbot/worker/package_macos64/build/src/builtins.c:705
#invokelatest#1 at ./essentials.jl:710 [inlined]
invokelatest at ./essentials.jl:709 [inlined]
run_interface at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/LineEdit.jl:2355
jfptr_run_interface_29039.clone_1 at /Applications/Julia-1.5.app/Contents/Resources/julia/lib/julia/sys.dylib (unknown line)
run_frontend at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:1144
#38 at ./task.jl:356
jfptr_YY.38_29911.clone_1 at /Applications/Julia-1.5.app/Contents/Resources/julia/lib/julia/sys.dylib (unknown line)
jl_apply at /Users/julia/buildbot/worker/package_macos64/build/src/./julia.h:1690 [inlined]
start_task at /Users/julia/buildbot/worker/package_macos64/build/src/task.c:705
Allocations: 28385145 (Pool: 28376481; Big: 8664); GC: 27
```

If the user uses a `;` it won't cause the seg fault.

**Comment by odow** [comment]
Reproducer is 
```Julia
using OSQP
import MathOptInterface; const MOI = MathOptInterface
model = OSQP.Optimizer()
MOI.get(model, MOI.NumberOfVariables())
```

**Comment by odow** [comment]
This can be closed as a duplicate of #10
